crates/forge/src/init.rs

```diff
@@ -212,7 +229,7 @@ pub fn run(project_name: &str) -> Result<()> {
     overwrite_files_from_scarb_template("src", &project_path, project_name)?;
     overwrite_files_from_scarb_template("tests", &project_path, project_name)?;
 
-    // Fetch to create lock file.
```

Comment:
Why is this comment not relevant anymore?

Reply:
Ill add the comment

Reply:
You added it back in the wrong place